src/components/Project/ProjectSelector.vue

```diff
@@ -1,20 +1,27 @@
-<!-- eslint-disable vue/require-prop-types -->
 <script setup>
 import { defineProps } from 'vue'
-defineProps(['project'])
+
+defineProps({
+	project: {
+		type: Object,
+		required: true
+	}
+})
 
 </script>
 
 <template>
   <v-expansion-panel>
     <v-expansion-panel-title
-      expand-icon="mdi-menu-down"
+      icon="chevron-down"
       class="projects_title"
     >
-      {{ project.projectTitle }}
+      {{ project[0].proposal_id }}
```

Comment:
is project an Object with entries like "0": {other obj}, or is it an array? In the props its defined as an object...

Reply:
it is an object, [0] is just accessing the first key so that we can get the proposal_id to display since all of the keys have the same proposal_id. I know it's not the finest approach, but I think for now it does what we want it to do. Here's an example of what project (in define props) looks like:

<img width="436" alt="Screenshot 2024-02-29 at 12 31 30 PM" src="https://github.com/LCOGT/datalab-ui/assets/54489472/96c238f8-bf0f-4341-8dc1-19e437a88163">

and here is one of the keys: 
<img width="447" alt="Screenshot 2024-02-29 at 12 31 58 PM" src="https://github.com/LCOGT/datalab-ui/assets/54489472/d3d08990-a564-4251-bd77-c88638edbbb8">

Reply:
You're right! I'm surprised I didn't get any warnings. Usually I get "expected ___ but got ___ instead". I updated it